person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+    @id = Random.rand(1..1000)
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  public
+
+  def can_use_services?
+    (is_of_age || @parent_permission)
+  end
```

Comment:
- [ ] Kindly check on your `can_use_services?` method, you have used `is_of_age` method instead of `of_age?` method

- [ ] [optional] @asnganje great work implementing the changes requested, however, it would have been better and easier just moving the `can use services` method to top off your private method. This makes the code more readable.